templates/reuse/dep5

```diff
@@ -1,5 +1,8 @@
+# SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+# SPDX-License: Unlicense
+
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
 Files: .gitlab/*
```

Comment:
What about `.github`?

Reply:
Well, that’s just an example of a dep5 file, and projects usually have only one of `{.github, .gitlab}`, so the choice is pretty much arbitrary. I think I just copy-pasted it from that morley PR.

Reply:
Ok, but isn't it weird that we have `.github` in templates and `.gitlab` in this file?

Reply:
Maybe 🤔. We can add a `.gitlab`template as well. 🤷‍♂